### ISSUE TITLE
SQLite bug

### ISSUE THREAD

**Issue by monday8am** [toggl/mobile#785]
XI: 8a185675667a461552b697a8a372934a99a2f0bd
System.InvalidCastExceptionCannot cast from source type to destination type.

```
at (wrapper castclass) object:__castclass_with_cache (object,intptr,intptr)
  at SQLite.SQLiteCommand+<ExecuteDeferredQuery>c__Iterator0`1[Toggl.Phoebe.Data.DataObjects.UserData].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Toggl.Phoebe.Data.DataObjects.UserData].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Toggl.Phoebe.Data.DataObjects.UserData]..ctor (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.ToList[UserData] (IEnumerable`1 source) [0x00000] in <filename unknown>:0 
```

**Comment by monday8am** [comment]
Pending of check. #794

**Comment by monday8am** [comment]
Closed in version 8.0.2
